packages/super-components/components/msc-billing-tile/docs/msc-billing-tile.stories.ts

```diff
@@ -0,0 +1,83 @@
+import { setupWorker } from 'msw';
+import { localeList, defaultLocale } from '@ovhcloud/msc-utils';
+import handlers from '../mock/handlers';
+
+setupWorker(...handlers).start({
+  onUnhandledRequest: 'bypass',
+  quiet: true,
+});
+
+export default {
+  title: 'Components/Manager Billing Tile',
+  tags: ['autodocs'],
+  render: ({
+    tileNumber = 1,
+    locale,
+    servicePath,
+  }: {
+    tileNumber: number;
+    locale: string;
+    servicePath: string;
+  }) => `
+    <section style="display: grid; grid-gap: 30px; grid-template-columns: repeat(3, 1fr);">
```

Comment:
**nitpick:** Instead of hard-coding the CSS in pixels, maybe we can think of creating a utility classes for super-components and reuse them to maintain consistency across the super-components.